roles/uchiwa/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+uchiwa_package_name: uchiwa
+uchiwa_service_name: uchiwa
+
+uchiwa_bind: 0.0.0.0
+uchiwa_port: 3000
+uchiwa_refresh: 5
+
+uchiwa_file_path: /etc/uchiwa/uchiwa.json
+uchiwa_sensu_api_server_name: uchiwa
+
+datacenters: [{"name": "{{ uchiwa_sensu_api_server_name }}","host": "{{ sensu_api_server }}","port": "{{ sensu_api_port }}" }]
```

Comment:
This should probably be uchiwa_datacenters, or sensu_datacenters, or something with a name prefix.